Project.toml

```
@@ -24,7 +24,7 @@ AbstractAlgebra = "^0.11.1"
 Antic_jll = "~0.2.2"
 Arb_jll = "~2.18.1"
 BinaryProvider = "0.4, 0.5"
-FLINT_jll = "~2.6.3"
+FLINT_jll = "~200.690.0"
 LoadFlint = "^0.3.4"
```

LoadFlint 0.4.0 is now in the registry, so we can do this:
```suggestion
LoadFlint = "^0.4.0"
```

---

With this it seems to work locally, but I don't have permission to push to this repo resp. this PR; so hopefully someone else can apply it.